recipes/lcms/all/conanfile.py

```diff
@@ -12,7 +16,7 @@ class LcmsConan(ConanFile):
     description = "A free, open source, CMM engine."
     license = "MIT"
     homepage = "https://github.com/mm2/Little-CMS"
-    topics = ("lcms", "cmm", "icc", "cmm-engine")
+    topics = ("little-cms", "cmm", "icc", "cmm-engine", "color-management-engine")
```

Comment:
Not sure how the search engine works for CCI, but I'd add `littlecms` as well, since that's how the jxl docs refer to it: https://fossies.org/linux/jpeg-xl/lib/jxl/docs/color_management.md#:~:text=source%20CMS%20is-,LittleCMS,-.%20It%20is%20also

Reply:
done